plugins/modules/source_control/gitlab/gitlab_project_variable.py

```diff
@@ -142,6 +187,45 @@
 from ansible_collections.community.general.plugins.module_utils.gitlab import auth_argument_spec, gitlab_authentication
 
 
+def vars_to_variables(vars, module):
+    # transform old vars to new variables structure
+    variables = list()
+    for item, value in vars.items():
+        if (isinstance(value, string_types) or
+           isinstance(value, (integer_types, float))):
+            variables.append(
+                {
+                    "name": item,
+                    "value": str(value),
+                    "masked": False,
+                    "protected": False,
+                    "variable_type": "env_var",
+                }
+            )
+
+        elif isinstance(value, dict):
+            new_item = {"name": item, "value": value.get('value')}
+
+            if value.get('masked'):
+                new_item['masked'] = value.get('masked')
+
+            if value.get('protected'):
+                new_item['protected'] = value.get('protected')
+
+            if value.get('environment_scope'):
+                new_item['environment_scope'] = value.get('environment_scope')
+
+            if value.get('variable_type'):
+                new_item['variable_type'] = value.get('variable_type')
+
+            variables.append(new_item)
```

Comment:
Shouldn't this be
```suggestion
            new_item = {
                "name": item,
                "value": value.get('value'),
                "masked": value.get('masked'),
                "protected": value.get('protected'),
                "variable_type": value.get('variable_type'),
            }

            if value.get('environment_scope'):
                new_item['environment_scope'] = value.get('environment_scope')

            variables.append(new_item)
```
so it behaves similar to the above case?